src/App.jsx

```diff
@@ -1,14 +1,77 @@
 import logo from "/logo.png";
 import "./App.css";
+import axios from "axios";
+import { useState } from "react";
+
+const WEATHER_API_KEY = "2e9b3bbe3e4c91069c9aef609fd54f11";
 
 function App() {
+  const [cityInputValue, setCityInputValue] = useState("");
+  const [currCity, setCurrCity] = useState("");
+  const [currTemp, setCurrTemp] = useState("");
+  const [weatherType, setWeatherType] = useState("");
+  const [weatherDesc, setWeatherDesc] = useState("");
+  const [weatherIcon, setweatherIcon] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    axios
+      .get(
+        `https://api.openweathermap.org/geo/1.0/direct?q=${cityInputValue}&limit=1&appid=${WEATHER_API_KEY}`
+      )
+      .then((response) => {
+        const cityGeoData = response.data[0];
+        return cityGeoData;
```

Comment:
You dont need to make a variable and then return it

.then((response) => response.data[0])
.then((cityGeoData) => ....
)